variables.tf

```diff
@@ -102,6 +102,20 @@ variable "ipv6_enabled" {
   nullable    = false
 }
 
+variable "disable_private_ipv6" {
+  type        = bool
+  description = "Set `true` to disable IPv6 addresses in private subnets"
+  default     = false
+  nullable    = false
+}
+
+variable "disable_public_ipv6" {
+  type        = bool
+  description = "Set `true` to disable IPv6 addresses in public subnets"
+  default     = false
+  nullable    = false
+}
+
```

Comment:
Would you be able to refactor these to use positive variable names to avoid double negatives? Variable inputs that enable/disable a setting should be formatted `...._enabled` (e.g. `encryption_enabled`).